test/conftest.py

```diff
@@ -256,6 +256,11 @@ def fixture_pghoard(db, tmpdir, request):
     yield from pghoard_base(db, tmpdir, request)
 
 
+@pytest.fixture(name="ipv6_webserver")
+def fixture_pghoard_ipv6_webserver(db, tmpdir, request):
```

Comment:
The fixture function name should be `fixture_` combined with the `name` property, so in this case I would probaly name the fixture `pghoard_ipv6` and the function `fixture_pghoard_ipv6`.

Reply:
:+1: